WcaOnRails/app/webpacker/components/EditSchedule/EditVenues/RoomPanel.js

```diff
@@ -26,13 +26,19 @@ function RoomPanel({
     }).then(() => dispatch(removeRoom(room.id)));
   };
 
+  const handleCopyRoom = () => {
+    dispatch(copyRoom(room.id));
+  };
+
   return (
     <Card fluid raised>
       <Card.Content>
         <Card.Header>
-          <Button floated="right" compact icon labelPosition="left" negative onClick={handleDeleteRoom}>
+          <Button floated="right" compact icon title="Remove" negative onClick={handleDeleteRoom}>
```

Comment:
Hm, too bad, it was working. I didn't really understand the `content` property. I've replaced `title` with a proper `Popup` though, which is doing exactly what I originally intended. I'd rather not use the name in the button since it's easy to overflow on smaller screens (but I'll change it to that if you want).

Reply:
Oh you wanted a _popup_? Sorry, that didn't occur to me... In that case, I'm torn between using officially documented SemUI props only, but on the other hand doing it this way feels much too bloated.

Reply:
Could animating the button be a compromise? https://react.semantic-ui.com/elements/button/#types-animated

Reply:
Isn't `title` the standard way to get a tooltip on a button?
Is the current `Popup` too bloated? We do this a lot already in the codebase (just search for `trigger={`).
Animated button could work, but it seems unnecessarily fancy.

Reply:
We do it even for simple buttons? In that case, fine :smile: